src/main/java/liquibase/ext/bigquery/database/BigqueryConnection.java

```diff
@@ -88,9 +88,6 @@ public Connection getUnderlyingConnection() {
     public void open(String url, Driver driverObject, Properties driverProperties) throws DatabaseException {
         if (driverProperties.stringPropertyNames().contains("Location")) {
             this.location = driverProperties.getProperty("Location");
-        } else {
-            this.location = getUrlParamValue(url, "Location", "US");
```

Comment:
But this way we will ignore the location from URL right? Maybe we need to replace everything by this:
```
if (driverProperties.stringPropertyNames().contains("Location")) {
            String locationValue = getUrlParamValue(url, "Location");
            driverProperties.setProperty("Location", locationValue);
        }
```

Reply:
That would make sense. That way the user can still specify the location if they want, otherwise, it will be the default to whatever the driver decides is best.